nautobot/dcim/templates/dcim/powerfeed.html

```diff
@@ -221,7 +221,7 @@ <h1>{% block title %}{{ object }}{% endblock %}</h1>
             </div>
             <div class="panel-body rendered-markdown">
                 {% if object.comments %}
-                    {{ object.comments|render_markdown }}
+                   {{ object.comments|render_markdown }}
```

Comment:
Looks like an unintended change?

Reply:
It was indeed. I thought I had checked each of them. Corrected now.